mitiq/rem/tests/test_rem.py

```diff
@@ -14,28 +14,30 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 """Unit tests for readout confusion inversion."""
-from cmath import isclose
+from typing import List
 from functools import partial
+import numpy as np
+import pytest
+
 import cirq
 from cirq.experiments.single_qubit_readout_calibration_test import (
     NoisySingleQubitReadoutSampler,
 )
-import numpy as np
-import pytest
 
-from mitiq.observable.observable import Observable
-from mitiq.observable.pauli import PauliString
-from mitiq import MeasurementResult
+from mitiq import MeasurementResult, Executor, Observable, PauliString
 from mitiq.rem.inverse_confusion_matrix import (
     generate_inverse_confusion_matrix,
 )
-from mitiq.rem.rem import execute_with_rem, mitigate_executor, rem_decorator
+from mitiq.rem import execute_with_rem, mitigate_executor, rem_decorator
 from mitiq.raw import execute as raw_execute
 from mitiq.interface.mitiq_cirq import sample_bitstrings
 
 # Default qubit register and circuit for unit tests
 qreg = [cirq.LineQubit(i) for i in range(2)]
-circ = cirq.Circuit(cirq.ops.X.on_each(*qreg), cirq.measure_each(*qreg))
+circ_with_measurements = cirq.Circuit(
+    cirq.ops.X.on_each(*qreg), cirq.measure_each(*qreg)
```

Comment:
Based on my other question, should we remove the measurements here, too?

Reply:
Since here we are not using the executor to evaluate an Observable but to just get the raw bitstrings, we need to explicitly add measurement gates to the circuit otherwise we get an empty result.